OpenRA.Game/Platform.cs

```diff
@@ -71,7 +71,7 @@ static string GetSupportDir()
 		{
 			// Use a local directory in the game root if it exists
 			if (Directory.Exists("Support"))
-				return "Support" + Path.DirectorySeparatorChar;
+				return Directory.GetCurrentDirectory() + Path.DirectorySeparatorChar + "Support" + Path.DirectorySeparatorChar;
```

Comment:
Better to use: 
return Path.Combine(GameDir, "Support") + Path.DirectorySeparatorChar;